Cargo.lock

```diff
@@ -462,7 +462,7 @@ checksum = "9b34d609dfbaf33d6889b2b7106d3ca345eacad44200913df5ba02bfd31d2ba9"
 [[package]]
 name = "asset-test-utils"
 version = "7.0.0"
-source = "git+https://github.com/moondance-labs/polkadot-sdk?branch=tanssi-polkadot-v1.11.0#1bbb680f1a54d4568e4c00e344bccaa61e926434"
+source = "git+https://github.com/moondance-labs/polkadot-sdk?branch=tanssi-polkadot-v1.11.0#6748a662c1d172b6c9aa6f559506fe0a0eb3f53a"
```

Comment:
This includes changes that we don't want in master, so undo changes to Cargo.lock, compile locally once and push